packages/design-system/src/images/usa-banner-lock.svg

```diff
@@ -1,3 +1,5 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 64">
+<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 64" aria-labelledby="banner-lock-title banner-lock-description" focusable="false">
+    <title id="banner-lock-title">Lock</title>
+    <desc id="banner-lock-description">A locked padlock</desc>
```

Comment:
I don't think we need to use `aria-labelledby` for the lock or dot-gov icon because they are purely decorative and are there to accompany the text

Reply:
I added it to stay consistent with USWDS which has `aria-labelleby` for `U.S flag` on the header and the `Lock` image on this content `Secure .gov websites use HTTPS` for screen readers

Reply:
And the `dot-gov` and other `lock` image is set with aria-hidden.

Reply:
I see what you mean Ni! Thanks! 
I think I agree with needing aria attributes for those images but I'm not going to block the merge. I'll bring this up with the USWDS team.

Reply:
@nichia [Same feedback as here](https://github.com/CMSgov/design-system/pull/916/files#r560486331). I'm ok with keeping the aria attributes but they should be handled in React not in the default SVG file